### PR TITLE
Brings in circleci to run tests

### PR DESCRIPTION
**Pull request by RobAWilkinson**
## Background
Brings in basic circleci setup to run tests.
 
## Definition of Done
 
- [ ] Pretty basic setup, nothing fancy so far.

### CODE REVIEW

**Comment by b00klegger** [comment]
👍 LGTM also. I don't have privs to `approve` it on the repo. I looked at the build in Circle and the config and *seems* to be OK. I haven't had much time to really look at the project (yet!).

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}-{{ checksum "yarn.lock" }}
+        
+      # Wait for DB to be available
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      # Database setup
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
+
+      # run tests!
+      - run:
```
Comment (hpjaj) [inline .circleci/config.yml]: Can you add RuboCop to these checks?

Comment (hpjaj) [reply .circleci/config.yml]: True dat, lint fails can be noisy.  But RuboCop failures are legit.  If a RuboCop failure happens in CI it is something that should trigger a failing build, and thereby has to be addressed before merging the PR to master.  Its great to have this protection backstopped at the CI level for ensuring code quality and consistency.

**Comment by RobAWilkinson** [comment]
@hpjaj @kfrz  I vote we merging this for now, and just figure out a sane rubocop and bringing in lint fails later in CI pipeline?

**Comment by hpjaj** [comment]
👍

**Pull request merged**